plugins/c-ares-dns-client.c

```diff
@@ -0,0 +1,757 @@
+/*
+ *
+ *  DNS Client - provides an asynchronous DNS resolution client.
+ *
+ *  The client is implemented using the c-ares library and integrated with
+ *  flimflam's glib main event loop. See http://c-ares.haxx.se and
```

Comment:
You might want to update this comment, as the integration is with ofono's glib main event loop, not flimflam's.